src/elements/content-sidebar/activity-feed/activity-feed/ActiveState.js

```diff
@@ -35,7 +29,7 @@ type Props = {
     getUserProfileUrl?: GetProfileUrlCallback,
     items: FeedItems,
     mentionSelectorContacts?: SelectorItems<>,
-    onAnnotationDelete?: ({ id: string, permissions?: BoxAnnotationPermission }) => void,
+    onAnnotationDelete?: (id: string) => void,
```

Comment:
Why are the permissions being removed?

Reply:
I don't think we need them in the call to delete. The permissions should apply whether you see the delete item / context menu but shouldn't be required as part of the call to delete. When I added this flow originally I modeled it after comment.

Reply:
That's probably true, but it might still be good to include in the api call -- there appears to be a `checkApiCallValidity` method used by the API classes which would have a trickle up(?) effect on the affected code. Developers might use the API classes directly like how we are for BoxAnnotations

Reply:
Re-added the permissions and we can implement them in the api call when we do the permissions slice of work.